gbench/syclblas/utils.hpp

```diff
@@ -23,7 +23,8 @@ namespace utils {
  * @brief Get the overall run time (start -> end) of a cl::sycl::event enqueued
  * on a queue with profiling.
  */
-inline cl_ulong time_event(cl::sycl::event e) {
+template<>
+inline cl_ulong time_event<cl::sycl::event>(cl::sycl::event& e) {
```

Comment:
if each function has a separate event type. you can use function overload instead of template. In this way the code will be more readable

Reply:
Discussed offline